client/dist/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Connect Four</title>
```

Comment:
You might want to consider changing your title to something more like... hmm, I don't know -- CAPBNB photos service ?

Reply:
Good call mate